cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -411,9 +411,8 @@ using scalar_device_type_t = typename type_to_scalar_type_impl<T>::ScalarDeviceT
 template <template <cudf::type_id> typename IdTypeMap = id_to_type_impl,
           typename Functor,
           typename... Ts>
-CUDA_HOST_DEVICE_CALLABLE constexpr decltype(auto) type_dispatcher(cudf::data_type dtype,
-                                                                   Functor f,
-                                                                   Ts&&... args)
+CUDA_HOST_DEVICE_FORCEINLINE_CALLABLE constexpr decltype(auto) type_dispatcher(
+  cudf::data_type dtype, Functor f, Ts&&... args)
```

Comment:
yeah, the formatting change here is all the proof I need to see that the current macro name is too long. At the very least we should drop the `CALLABLE` part.

Reply:
@bdice @jrhemstad So are we happy going with `CUDA_HOST_DEVICE_FORCEINLINE` then? Or perhaps we even drop the `CUDA`? I can run with any version of this that we're all happy with. There may be some dissonance with the definition of `CUDA_HOST_DEVICE_CALLABLE` if the names don't entirely match up, though, so that's the other tradeoff. I'm still not convinced that an actual abbreviated version will be clear enough to be useful, and it's hard to know where to find the definition of a macro like this when everything includes types.hpp, but I'm fine being voted down there if you both end up on the side of something concise like `CUDA_HDFI` (or `CUDA_HDFI_CALLABLE`).

Reply:
Just saw your message on the other thread. Going to resolve this conversation so we can just decide there.